src/routes/Edit/VideoSplitter.tsx

```diff
@@ -123,6 +128,26 @@ const VideoSplitter = ({
     end && updateSegmentAt(index, { end });
   };
 
+  const setDefaultTimeSpan = () => {
+    const start = timeFormat.from(String(duration * 0.25));
+    start && updateSegmentAt(index, { start });
+    const end = timeFormat.from(String(duration * 0.75));
+    end && updateSegmentAt(index, { end });
+  };
+
+  const setSegmentEndTime = () => {
+    let pauseTime = String((window as any).$gsecs);
+    const end = timeFormat.from(pauseTime);
+    //console.log(end, currentSegment?.end , currentSegment?.start);
```

Comment:
If possible lets try to clean up/remove any commented log lines.

Reply:
Done...